tests/unit/CoreGroupTest.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace WPGraphQL\ContentBlocks\Unit;
+
+/**
+ * @group block
+ * @group core-group
+ */
+final class CoreGroupTest extends PluginTestCase {
+
+	/**
+	 * The ID of the post created for the test.
+	 *
+	 * @var int
+	 */
+	public $post_id;
+
+	/**
+	 * The ID of the attachment created for the test.
+	 *
+	 * @var int
+	 */
+	public $attachment_id;
+
+	public function setUp(): void {
+		parent::setUp();
+
+		$this->post_id = wp_insert_post(
+			[
+				'post_title'   => 'Post Title',
+				'post_content' => '',
+				'post_status'  => 'publish',
+			]
+		);
+
+		\WPGraphQL::clear_schema();
+	}
+
+	public function tearDown(): void {
+		// your tear down methods here
+		wp_delete_post( $this->post_id, true );
+		\WPGraphQL::clear_schema();
+
+		parent::tearDown();
+	}
+
+	/**
+	 * Get the query for the CoreGroup block.
+	 *
+	 * @param string $attributes The attributes to add to query.
+	 */
+	public function query(): string {
+		return '
+			fragment CoreGroupFragment on CoreGroup {
+			  attributes {
+			    align
+			    backgroundColor
+			    borderColor
+			    className
+			    cssClassName
+			    fontFamily
+			    fontSize
+			    gradient
+			    layout
+			    lock
+			    style
+			    tagName
+			    textColor	
+			  }
+			}
+			
+			query Post($id: ID!) {
+			  post(id: $id, idType: DATABASE_ID) {
+			    databaseId
+			    editorBlocks(flat:true) {
+			      apiVersion
+			      blockEditorCategoryName
+			      clientId
+			      cssClassNames
+			      innerBlocks {
+			        name
+			      }
+			      name
+			      parentClientId
+			      renderedHtml
+			      type
+			      ...CoreGroupFragment
+			    }
+			  }
+			}
+		';
+	}
+
+	/**
+	 * Test that the CoreGroup block is retrieved correctly.
+	 *
+	 * Covers the following attributes:
```

Comment:
(this isn't a list of attributes. 
Not saying you need one - we added it to track what Core*Attributes are actually being tested since this project doesn't have accurate codecov )

Reply:
Thanks @justlevine 

I will update the comment and that's good to know about the code coverage.

Also thanks for the code review 👍